crossing-training.lic

```diff
@@ -243,6 +248,32 @@ class CrossingTraining
     bput("put 5 silver #{currency} in almsbox", 'You drop', 'But you do not', 'attend to thy own woes')
   end
 
+  def check_chadatru
+    return unless DRStats.paladin? && @pray_to_chadatru
+    return unless Time.now - UserVars.chadatru_prayer_last > @chadatru_prayer_cooldown
+
+    case @hometown
+    when 'Crossing'
+      walk_to('5844')
```

Comment:
This feels like some data that should be in `base-hometown` or `base-theurgy` (so you don't need a big case/when block). What do you think?

Reply:
Probably would be a good idea. I can do that.